### PR TITLE
Fixes and updates:

### LINKED ISSUE #11: Jumpsuits cover mouths.
That's it. I think it's a problem with modifications to limb defines that were made in order to bring back foot - leg gradation. Leaving it here to not forget to fix.

### PR DESCRIPTION
**Pull request by drexample**
Fixes jumpsuits and suits covering face
Fixes bodypart selection doll missing iconstates
Fixes phoronglass missing a sprite
Fixes body modification menu missing entry for torso
Fixes phantom head overlaying your actual when using prosthetics
Removes that one warning in configuration_eclipse.dm
Removes unused duplicate file
Removes heavy PvP oriented storyteller from rotation
Makes storytellers less batshit insane
Adds back ckey whitelist with discord redirect

Closes #11

### CODE REVIEW

File: code/controllers/configuration_eclipse.dm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 		var/pos = findtext(t, " ")
 		var/name = null
 		var/value = null
-
+		value = value
```
Comment (EvilJackCarver) [inline code/controllers/configuration_eclipse.dm:32]: does this actually suppress the compile warning

Comment (drexample) [reply code/controllers/configuration_eclipse.dm:32]: yeah

Comment (drexample) [reply code/controllers/configuration_eclipse.dm:32]: изображение

Comment (EvilJackCarver) [reply code/controllers/configuration_eclipse.dm:32]: that's feckin' amazing

File: code/_helpers/global_lists.dm [hunk 2]
```diff
@@ -1,238 +0,0 @@
-var/list/clients = list()							//list of all clients
```
Comment (EvilJackCarver) [inline code/_helpers/global_lists.dm:1]: Unused file?

File: code/game/jobs/manifest_controller.dm [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+/datum/manifest_controller
+	var/active = 1
+
+/datum/manifest_controller/New()
+	spawn(3 MINUTES) //Lots of lag at the start of a shift.
+		process()
+
+/datum/manifest_controller/proc/process()
+	update_manifest()
+	load_aeiouwhitelist()
```
Comment (EvilJackCarver) [inline code/game/jobs/manifest_controller.dm:14]: Only thing I'm seeing is the manifest controller and the whitelist code both reference aeiou

Up to you if you want to fix it before you merge, or leave it in as an artifact easter egg or whatever

File: code/game/jobs/whitelist.dm [hunk 4]
```diff
@@ -22,3 +22,103 @@ var/list/whitelist = list()
...
 
 #undef WHITELISTFILE
+
+
+
+var/list/aeiou_whitelist = list()
+var/list/blacklist = list()
+
+/hook/startup/proc/loadaeiouwhitelist()
+	load_aeiouwhitelist()
```
Comment (EvilJackCarver) [inline code/game/jobs/whitelist.dm:32]: See previous note about aeiou

File: code/game/jobs/manifest_controller.dm (hunk 3 above)
Comment (drexample) [reply code/game/jobs/manifest_controller.dm:14]: aeiouwhitelist is technically name of the mechanic

File: code/_helpers/global_lists.dm (hunk 2 above)
Comment (drexample) [reply code/_helpers/global_lists.dm:1]: a duplicate, caused by git clients not recognizing lowercase and uppercase

Comment (drexample) [reply code/_helpers/global_lists.dm:1]: Was not included in the DME or used anywhere else

File: code/game/jobs/manifest_controller.dm (hunk 3 above)
Comment (EvilJackCarver) [reply code/game/jobs/manifest_controller.dm:14]: that's fair, just wanted to make a note of it

**Pull request merged**